config/config.reference.hocon

```diff
@@ -66,7 +66,6 @@
         "bqWriteRequestTimeout": "1 second"
```

Comment:
Looks like I reviewed this commit in the wrong branch, but you can find my comments [here](https://github.com/snowplow-incubator/snowplow-bigquery-loader/pull/277).